app/src/features/InntektsmeldingSkjemaState.tsx

```diff
@@ -0,0 +1,52 @@
+import type { ReactNode } from "@tanstack/react-router";
+import type { Dispatch, SetStateAction } from "react";
+import { createContext, useContext, useState } from "react";
+
+export type InntektsmeldingSkjemaState = {
+  kontaktperson?: { navn: string; telefon: string };
+  korrigertMånedslønn?: number;
+  skalRefunderes?: boolean;
+  refusjonsbeløpPerMåned?: number;
+  endringIRefusjon?: boolean;
+  refusjonsendringer?: { måned: string; beløp: number }[];
+  misterNaturalytelser?: boolean;
+  naturalytelserSomMistes?: { navn: string; beløp: number; fraOgMed: string }[];
+};
```

Comment:
Åpen for kommentarer på denne delen – er det gode nok navn, og er det ting jeg har glemt?

Reply:
De propertyene som er array trenger kanskje ikke være optional? Og heller ha [] som default verdi?